core/templates/dev/head/components/ExplorationCreationButtonService.js

```diff
@@ -140,31 +140,38 @@ oppia.factory('ExplorationCreationButtonService', [
       showCreateExplorationModal: function(categoryList) {
         alertsService.clearWarnings();
 
-        siteAnalyticsService.registerOpenExplorationCreationModalEvent();
-
-        getModalInstance(categoryList, false).result.then(function(result) {
-          var category = $filter('normalizeWhitespace')(result.category);
-          if (!validatorsService.isValidEntityName(category, true)) {
-            return;
-          }
+        var currentPage = window.location.pathname;
+
+        if (currentPage !== '/my_explorations' &&
+          currentPage !== '/my_explorations?mode=create') {
```

Comment:
Doesn't the pathname generally exclude the query string?

     https://developer.mozilla.org/en-US/docs/Web/API/Location